app/controllers/pair_requests_controller.rb

```diff
@@ -13,6 +13,7 @@ def show
   end
 
   # POST /pair_requests or /pair_requests.json
+  # rubocop:disable Metrics/AbcSize
```

Comment:
I ignored the Assignment Branch Condition size lint error here -- I think the code here is probably fine, but I don't want to disable in general.